third-party/realdds/include/realdds/dds-stream-server.h

```diff
@@ -186,4 +188,19 @@ class dds_pose_stream_server : public dds_motion_stream_server
 };
 
 
+class dds_metadata_stream_server : public dds_stream_server

```

Comment:
If this is a stream, it's part of the stream headers etc. that's part of device initialization?

If so, why? Why not an implicit topic that can go along with every other stream? Like a `notification`?

Reply:
I implemented as a stream based on our initial discussion of this topic, you said that metadata is just another stream of information and should be handled like other streams of the device.
On our subsequent discussions I mentioned that it is different, because it has no profiles, and that I need some logic to differentiate and handle the initialization.

Will consider changing in future PRs

Reply:
"Will consider changing" is like saying "some day" which is like saying "never" :)

Reply:
Implemented as a single topic of the device (not of type `dds_stream/_server`)